--- tealdeer-1.7.1/Cargo.toml	1970-01-01T00:00:01+00:00
+++ tealdeer-1.7.1/Cargo.toml	2024-12-10T15:43:40.365655+00:00
@@ -102,7 +102,7 @@
 version = "1"
 
 [dependencies.zip]
-version = "2.1.6"
+version = ">=0.6,<3"
 features = ["deflate"]
 default-features = false
 
@@ -122,7 +122,7 @@
 version = "3.1.0"
 
 [features]
-default = ["native-roots"]
+default = ["native-tls"]
 logging = ["env_logger"]
 native-roots = ["reqwest/rustls-tls-native-roots"]
 native-tls = ["reqwest/native-tls"]
